images/cellregmap/run_interaction.py

```diff
@@ -0,0 +1,219 @@
+import os
+import sys
+
+import click
+import scanpy as sc
+import pandas as pd
+import xarray as xr
+from numpy import ones
+from pandas_plink import read_plink1_bin
+from numpy.linalg import cholesky
+from limix.qc import quantile_gaussianize
+
+from cellregmap import run_interaction
+
+
+@click.command()
+@click.option("--chrom", required=True)
+@click.option("--gene-name", required=True)
+@click.option("--sample-mapping-file", required=True)
+@click.option("--genotype-file", required=True)
+@click.option("--phenotype-file", required=True)
+@click.option("--context-file", required=True)
+@click.option("--kinship-file", required=True)
+@click.option("--feature-variant-file", required=True)
+@click.option(
+    "--output-folder", required=False, default=""
+)  # by default current directory, where you are running your script from
+@click.option("--n-contexts", required=False, type=int)
+
+def main(
+    chrom,
```

Comment:
Not super important, but adding type annotations could be useful, eg:

```suggestion
    chrom: str,
```